src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
+				var dropdowntag = dropdown.tagName;
+				if (dropdowntag === 'DATALIST') {
+					var inputsibling = dropdown.previousElementSibling;
```

Comment:
@tchaffee @tbushman Important to note that this stackoverflow questions seems to be pre ES6 and the reasoning for not defining variables inside if statements is based on the lack of block scoping in ECMAScript. However with ES6's `let` keyword, we can now properly block scope variables to `if` statements and other code blocks. In general, this should be ok. The way we have it now is fine though.